packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionContainerStart.test.ts

```diff
@@ -5,15 +5,15 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { Benny, getFunctionsBinary } from '@heroku/functions-core';
+import { getFunctionsBinary } from '@heroku/functions-core';
 import * as library from '@heroku/functions-core';
 import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode/out/src';
 import { EventEmitter } from 'events';
 import * as path from 'path';
 import { assert, createSandbox, SinonSandbox, SinonStub } from 'sinon';
 import { Uri, window } from 'vscode';
 import { channelService } from '../../../../src/channels';
-import { forceFunctionStart } from '../../../../src/commands/functions/forceFunctionStart';
+import { forceFunctionContainerStartCommand } from '../../../../src/commands/functions/forceFunctionContainerStartCommand';
```

Comment:
This is testing forceFunctionContainerStartCommand, so this file should be named forceFunctionContainerStartCommand.test.ts (not forceFunctionContainerStart.ts)